src/yafowil/common.py

```diff
@@ -1783,6 +1804,11 @@ def number_extractor(widget, data):
     return val
 
 
+@managedprops('min', 'max', 'step', *managed_props__input_full)
+def number_edit_renderer(widget, data):
+    return input_generic_renderer(widget, data)
+
```

Comment:
No need for dedicated ``number_edit_renderer``. ``min`` ``max`` and ``step`` managed props are defined for ``number_extractor``, and managed props gets aggregated